datastore_entity_derives/src/lib.rs

```diff
@@ -234,6 +234,19 @@ pub fn datastore_managed(input: TokenStream) -> TokenStream {
                 #kind_str
             }
         }
+
+        impl datastorers::DatastorersQueryable<#name> for #name {
```

Comment:
Couldn't this maybe be a blanket impl instead of proc macro?

Just guessing but something like

```rust
impl <T> datastorers::DatastorersQueryable<T> for T
    where T: Kind + TryFrom<DatastoreEntity, Error = DatastorersError>
```

maybe?

Reply:
ah! yes, probably true, that would be good, but would probably not work if we add an enum for the properties, but will change to blanket for now

Reply:
required an extra trait to get the page size from the macro, but that was solvable, will be part of next push, almost there